hack/release.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/release.sh
+
+# Set default GCS/GCR
+: ${BUILD_RELEASE_GCS:="knative-releases/build-pipeline"}
```

Comment:
I think it would be better to use BUILD_PIPELINE_RELEASE_GCS/BUILD_PIPELINE_RELEASE_GCR to avoid issues with the ones used by knative/build.